cmake/CMakeLists.txt

```diff
@@ -71,11 +71,13 @@ endif()
 
 configure_file(OpenEXRConfig.h.in ${CMAKE_CURRENT_BINARY_DIR}/OpenEXRConfig.h)
 configure_file(OpenEXRConfigInternal.h.in ${CMAKE_CURRENT_BINARY_DIR}/OpenEXRConfigInternal.h)
+configure_file(../src/lib/OpenEXRCore/OpenEXRVersion.h ${CMAKE_CURRENT_BINARY_DIR}/OpenEXRVersion.h COPYONLY)
 
 if(OPENEXR_INSTALL)
 install(
   FILES
     ${CMAKE_CURRENT_BINARY_DIR}/OpenEXRConfig.h
+    ${CMAKE_CURRENT_BINARY_DIR}/OpenEXRVersion.h
```

Comment:
couldn't this just be ${CMAKE_PROJECT_SOURCE_DIR}/src/lib/OpenEXRCore/OpenEXRVersion.h?

Reply:
ah yes